packages/medusa-plugin-sendgrid/src/services/sendgrid.js

```diff
@@ -615,14 +616,15 @@ class SendGridService extends NotificationService {
       {
         id: returnRequest.items.map(({ item_id }) => item_id),
       },
-      { relations: ["tax_lines"] }
+      { relations: ["tax_lines", "variant", "variant.product"] }
```

Comment:
@riqwan - check if product is needed here.

Reply:
comment: I don't think it is needed. I believe we can remove all `relations` and only `select` id to make the query more performant

Reply:
Items are only used on line 639 as far as I can see

Reply:
changing any of these breaks the existing snapshot. Keeping this as is to not change the outcome of the response.

Reply:
Would it make sense to update the snapshot? `items` are not included in the returned value of the method, so shouldn't affect anything

Reply:
It is in the return value. Each `item` in that array is modified in `returnItems` in line 639. and `returnItems` are returned in the response of the method.

Reply:
Ah my bad, I missed that 👍